# Challenge_6_exception
Starting with a README file
